scipy/stats/_hypotests.py

```diff
@@ -130,3 +133,224 @@ def epps_singleton_2samp(x, y, t=(0.4, 0.8)):
     p = chi2.sf(w, r)
 
     return Epps_Singleton_2sampResult(w, p)
+
+
+cvm_testResult = namedtuple('cvm_testResult', ('statistic', 'pvalue'))
+
+
+def _psi1_mod(x):
+    """
+    psi1 is defined in equation 1.10 in Csorgo, S. and Faraway, J. (1996).
+    This implements a modified version by excluding the term V(x) / 12
+    (here: _cdf_cvm_inf(x) / 12) to avoid evaluating _cdf_cvm_inf(x)
+    twice in _cdf_cvm.
```

Comment:
This is difficult to compare against Csorgo and Faraway 1.10 directly. Can you offer any insight that will make it easier to do, or should I just try to review against the Maple code? Was that Maple code distributed with the paper?

Reply:
Hi Matt, the Maple code is accessible on Julian Faraway's website: https://people.bath.ac.uk/jjf23/papers/index.html, the direct link is in the PR description above

And yes, it is best to review the code for psi_mod against the MAPLE version.

I will get back to the second question

Reply:
> Also, since the V(x) / 12 is removed here, is it the line: y[sup] = _cdf_cvm_inf(x[sup]) * (1 + 1./(12*n)) + _psi1_mod(x[sup]) / n that adds it back with the 1./(12*n)?

yes

Reply:
Thanks. Unfortunately, I need to put my review on hold for now : / Thanks for your patience.